packages/access-api/wrangler.toml

```diff
@@ -25,6 +25,10 @@ binding = "__D1_BETA__"
 database_name = "spaces-dev"
 database_id = "7c676e0c-b9e7-4711-97c8-7b1c8eb229ae"
 
+[[r2_buckets]]
```

Comment:
Assuming this binding name works for everyone, I will need to add a corresponding different bucket_name for each of env.staging, env.production